codice/alliance
# [0.3.x] CAL-375 Updated the DefaultSecurityAttributeValuesPlugin to use the system high values from the etc/users.attributes file

#### What does this PR do?
This PR updates the DefaultSecurityAttributeValuesPlugin to grab the system high attribute values from the users.attributes file instead of the user's claims.
#### Who is reviewing it? 
@peterhuffer @AzGoalie @oconnormi @Lambeaux @ryeats
#### Choose 2 committers to review/merge the PR.
@clockard
@lessarderic
#### How should this be tested?
This PR is dependent on https://github.com/codice/ddf/pull/2695.
Confirm that metacard security attributes are updated as expected on ingested metacards when the `DefaultSecurityAttributeValuesPlugin` configuration is updated.
#### What are the relevant tickets?
[CAL-375](https://codice.atlassian.net/browse/CAL-375)
#### Checklist:
